search/config.py

```diff
@@ -0,0 +1,192 @@
+"""
+Flask configuration.
+
+Docstrings are from the `Flask configuration documentation
+<http://flask.pocoo.org/docs/0.12/config/>`_.
+"""
+import os
+
+ON = 'yes'
+OFF = 'no'
+
+DEBUG = os.environ.get('DEBUG') == ON
+"""enable/disable debug mode"""
+
+TESTING = os.environ.get('TESTING') == ON
+"""enable/disable testing mode"""
+
+PROPAGATE_EXCEPTIONS = \
+    True if os.environ.get('PROPAGATE_EXCEPTIONS') == ON else None
+"""
+explicitly enable or disable the propagation of exceptions. If not set or
+explicitly set to None this is implicitly true if either TESTING or DEBUG is
+true.
+"""
+
+PRESERVE_CONTEXT_ON_EXCEPTION = \
+    True if os.environ.get('PRESERVE_CONTEXT_ON_EXCEPTION') == ON else None
+"""
+By default if the application is in debug mode the request context is not
+popped on exceptions to enable debuggers to introspect the data. This can be
+disabled by this key. You can also use this setting to force-enable it for non
+debug execution which might be useful to debug production applications (but
+also very risky).
+"""
+
+SECRET_KEY = os.environ.get('SECRET_KEY', 'asdf1234')
```

Comment:
Should the comment indicate where to set this environment variable so that it gets picked up in production configuration?

Reply:
This will be an environment variable, so how/where it gets set will depend on the deployment context. Good reminder to add deployment documentation.

That said, we're not actually using this particular config variable, so it can probably be removed.

Reply:
This is removed in #108